windows-vm-configure-winrm/azuredeploy.json

```diff
@@ -0,0 +1,244 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "newStorageAccountName": {
```

Comment:
This is going to cause a failure. It's best to move the storageaAccount name to a variable. Autogenerate it like below. See the proposal in https://github.com/Azure/azure-quickstart-templates/issues/944#issuecomment-166189871
concat(uniqueString() + 'sa' + <scenario name>)